tests/unittest/llmapi/test_llm.py

```
@@ -1980,6 +1990,45 @@ async def task():
     asyncio.run(task())
 
 
+class DummyExecutorWorker4(GenerationExecutorWorker):
+    should_raise_error = True
+
+    def submit(self, request: GenerationRequest) -> GenerationResult:
+        # Making the first response failed, and the subsequent responses successful
+        if DummyExecutorWorker3.should_raise_error:
+            DummyExecutorWorker3.should_raise_error = False
+            raise RequestError("Test error")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)